src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectIssueResource.java

```diff
@@ -252,6 +258,25 @@ public Response addLabel(@FormParam("tag") String tag, @FormParam("color") Strin
 		);
 		return redirect(new URI(request.getRequestURI()).resolve("issues"));
 	}
+	
+	@POST
+	@Path("deleteLabel/{labelId}")
```

Comment:
According to REST conventions it is better to create the endpoints:

```java
@POST
@Path("label")
```

And:
```java
@DELETE
@Path("label/{labelId}")
```

Then in JQuery you would do:
```js
  return $.ajax({
    url: url,
    type: 'DELETE',
    success: callback,
    data: data,
    contentType: type
  });
```

See also http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api
And http://stepansuvorov.com/blog/2014/04/jquery-put-and-delete/